content/docs/1.2.4/volumes-and-nodes/volume-size.md

```diff
@@ -19,29 +19,81 @@ In this section, you'll have a better understanding of concepts related to volum
 - Since all historical data stored in the snapshots and active data will be calculated into the actual size, the final value can be greater than the nominal size.
 - The actual size will be shown only when the volume is running.
 
-## An example that helps understand volume `Size` and volume `Actual size`:
+## Example
 
-Here, we will have an example that explains how volume `size` and `actual size` get changed after a bunch of I/O and snapshot related operations.
+In the example, we will explain how volume `size` and `actual size` get changed after a bunch of IO and snapshot related operations.
 
-{{< figure src="/img/screenshots/volumes-and-nodes/volume-size-example-0.png" >}}
 > The illustration presents the file organization of **one replica**. The volume head and snapshots are actually sparse files, which we mentioned above.
 
-1. Create a 5Gi volume, then attach and mount it on a node. See Figure 1 of the illustration.
-    - For this empty volume, the nominal `size` is 5Gi and the `actual size` is almost 0.
-    - There is some meta info in the volume hence the `actual size` is not exactly 0.
-2. Write 2Gi data (data#1) in the volume mount point and take a snapshot (snapshot#1). See Figure 2 of the illustration.
-    - Now data#1 is stored in snapshot#1 and the `actual size` is 2Gi.
-{{< figure src="/img/screenshots/volumes-and-nodes/volume-size-example-1.png" >}}
-3. Delete data#1 from the mount point.
-    - The truth of the data#1 deletion is that the data#1 is marked as removed in **the filesystem level** (For example, inode deletion in ext4). Since **Longhorn operates on the block level** and does not understand the filesystem, as a result, the disk blocks/space storing data#1 won't be released after the deletion.
+{{< figure src="/img/screenshots/volumes-and-nodes/volume-size-illustration.png" >}}
```

Comment:
NIT: You can replace the description of the gray part with `the blocked freed by the file deletion of filesystem but still considered as allocated in Longhorn's perspective`